raffle.py

```diff
@@ -10,22 +11,26 @@
         if row['What username do you go by online?'] == '-':
             username = row['Ticket First Name']
         winlist.append(row['Number'] + " " +  username)
-[print(thing) for thing in random.sample(winlist, len(winlist))[0:30]]
-slide = open('slide.html','w')
+[print(thing) for thing in random.sample(winlist, len(winlist))[0:totalWinners]]
+slide = open('winners.html','w')
 slide.write("""
- <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>SMK Winter Meetup 2018 Winners!</title>
-    <h1>And the winners are...</h1>
-  </head>
-  <body>
+<!doctype html>
+<html lang=en>
+<head>
+<meta charset=utf-8>
+<title>SMKmeetup Giveaway Winners!</title>
+<link rel="stylesheet" href="raffle-style.css">
+</head>
+<body>
+<h2>SMKmeetup Giveaway Winners</h2>
+<ul class="multi-12">

```

Comment:
We should probably use Jinja templates or something for this instead of building raw strings

Reply:
Jinja what? 😜

Reply:
http://jinja.pocoo.org/